roles/taca/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+- name: Fetch TACA-ngi-pipeline from GitHub
+  git: repo={{ taca_ngi_repo }}
+       dest={{ taca_ngi_dest }}
+       version={{ taca_ngi_version }}
+       force=yes
+  tags: taca
+
+- name: Install TACA-ngi-pipeline requirements
+  shell: /lupus/ngi//sw/anaconda/envs/NGI/bin/pip install -r /lupus/ngi/sw/taca-ngi-pipeline/requirements.txt
+  tags: taca
+
+- name: Install TACA-ngi-pipeline
+  shell: cd {{ taca_ngi_dest }} && /lupus/ngi/sw/anaconda/envs/NGI/bin/pip install .
```

Comment:
Same thing here as on L9. 

You could also split out the module arguments like: 

```
shell: (...)pip install 
args: 
  chdir: {{ taca_ngi_dest }}
```

to make it a bit more readable and easy to add more arguments in the future.